src/mergerfs.dedup

```diff
@@ -118,16 +118,23 @@ def noop_dedup(stats):
 def manual_dedup(stats):
     done = False
     while not done:
+        value = input('Which is keep?: ')
+
+        if value.lower() in ('skip', ''):
```

Comment:
Why bother with 'skip' if hitting enter will do the same? I'm okay with "enter to skip" but the input line and errors should include that fact. 

Or make it "Which to keep? ('s' to skip):